src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const employeeList = [...document.querySelectorAll('li')];
+const mainList = document.querySelector('ul');
+
+function salarySort(list) {
+  list.sort((a, b) => (+b.dataset.salary.slice(1).split(',').join(''))
+   - (+a.dataset.salary.slice(1).split(',').join('')));
+
+  return list;
+}
+
+function sortList(list) {
+  mainList.append(...salarySort(list));
+}
+
+function getEmployees(list) {
+  const employees = [];
+  const copyList = [...list];
+
+  for (const item of copyList) {
+    employees.push({
+      name: item.innerText,
+      position: item.dataset.position,
+      salary: item.dataset.salary,
+      age: item.dataset.age,
+    });
+  }
+
+  return employees;
+}
```

Comment:
why do you have this function? I mean: you create some function that returns new array, and then where do you need that array?

Reply:
Hi, Tetiana! Thanks for review!

I just thought that second function must create exactly new array (I don't know why I thought it)) )

If I have understood you in the right way, the second function must work with the previous list and I don't have to create a new array. I have done it, check please!